### ISSUE TITLE
[Feature] Show note icon when account has notes

### ISSUE THREAD

**Issue by MikeBishop** [actualbudget/actual#691]
### Verified feature request does not already exist?

- [X] I have searched and found no existing issue

### 💻

- [ ] Would you like to implement this feature?

### Pitch: what problem are you trying to solve?

When a budget category has a note, the note icon is always visible, like this:
![image](https://user-images.githubusercontent.com/4273797/221274664-34ec8b94-4ad9-49b9-9f30-cd3c9194e2cb.png)

Account notes were recently added, but the icon only appears on hover even if there is a note present, like this:
![image](https://user-images.githubusercontent.com/4273797/221274783-80049ead-0451-4351-9ea8-4dc0bf520b8a.png)
![image](https://user-images.githubusercontent.com/4273797/221274823-1a8b7667-3f31-4b46-9347-ffe0a4abfb69.png)

### Describe your ideal solution to this problem

Account notes should behave the same way category notes do, displaying the icon all the time if the note is non-empty.

**Comment by j-f1** [comment]
I believe this already happens — can you send a screen recording showing the issue?

<img width="194" alt="Screenshot_2023-02-24 15 42 18" src="https://user-images.githubusercontent.com/25517624/221287483-7cfc47b2-e90f-4b8c-82b7-2a5ee30307db.png">

**Comment by Kidglove57** [comment]
I confirm that I cannot see that there is a Note to an account until the mouse is right over that area (right of the account name). Only then it shows both the edit button and the notes button.

**Comment by j-f1** [comment]
You might be on an old version, then — the current version of Actual (`v23.2.9`) has the icon always visible:

https://user-images.githubusercontent.com/25517624/221312508-f064d04f-3a39-4513-a3f8-3e16432becbe.mov

**Comment by Kidglove57** [comment]
This is indeed very odd. i have double checked - I am on v 23.2.9!
I have tested as follows on my Mac and on my iPad:
1) In Firefox and Safari
2) On the version I have hosted on PikaPods and on that on Fly.
No Accounts note or edit button appears until I move the mouse to that area and then the button does appear and the content of the note is also revealed on hover, as expected.

**Comment by shall0pass** [comment]
I was seeing the same effect @Kidglove57 saw today using the master netlify site, brave browser, windows 10.  I don't see the same behavior in brave on Android.  I'll see if I can repro it later.

**Comment by Kidglove57** [comment]
@MatissJanis @shall0pass just to confirm that all my accounts have notes and I am still not seeing the accounts notes button until I move the mouse to that area.

**Comment by gsumpster** [comment]
@Kidglove57, can you shoot over the version of Safari that you're seeing this on? I'm unable to reproduce on 16.2 on my Mac, even with notes on all accounts

**Comment by Kidglove57** [comment]
@gsumpster thanks! I have Safari 16.3. However, the Accounts note icon is not visible for me on my MacBook Air or my iPad Air in Safari, Firefox, Brave, Chrome.

BUT interestingly on setting up a new account and note in the preview version (Netlify) I can now see the note icon!

I have tried adding a brand new account and note in my production version of Actual - but still no account icon showing.

**Comment by MatissJanis** [comment]
Are you able to reproduce the issue here? https://actualbudget.netlify.app/

If yes: could you provide reproduction steps?

**Comment by Kidglove57** [comment]
@MatissJanis  No. I confirm that I cannot reproduce the issue at  https://actualbudget.netlify.app/

However, I have tested my two production files (PikaPods and Fly hosted) for Actual on every browser I have on my MacBook Air (Firefox, Safari, Edge, Brave and Chrome), iPad Air and iPhone (with populated accounts notes). The notes icon does not show on any of them until I hover over that area.

I'll be interested to hear what @MikeBishop and @shall0pass report.

**Comment by shall0pass** [comment]
I just used the netlify link and was able to reproduce the problem.  I linked it to my personal actual-server instance to pull my budget.
![note_missing2](https://user-images.githubusercontent.com/20625555/221383268-88dcc0d4-c822-45b3-98d8-fe315a6d83b2.gif)

I then logged out of my server and loaded a test budget.  Added "Ally" to the note and tried.  This was the result.

![note_missing3](https://user-images.githubusercontent.com/20625555/221383342-79ef5867-b565-4fc8-bdb9-20f8308682cf.gif)

**Comment by shall0pass** [comment]
I'll add.  I cannot consistently repeat this behaviour.  I tried using Edge, and got the expected results where the icon is visible when text is present.  I couldn't even repeat the bahaviour this morning using the same browser on the same computer on the same netlify link.  I don't have any insights into what is different beside AM or PM.

**Comment by Kidglove57** [comment]
I have experimented further with my real life file. 
1) I am still NOT able to see the accounts notes icon when I launch Actual from within any of the browsers that I mentioned earlier
2) I CAN see the notes icon at https://actualbudget.netlify.app/. This is true with a test budget and also when I import my main "production" file into Netlify.
But this is where it gets puzzling.
3) If I drag the Actual link from Safari to the dock and launch Actual from the dock I CAN see the Notes icon!
4) However, if I drag the link to the dock from any browser other than Safari I CANNOT see the Notes icon on launch.
5) Finally, I disabled each browser extension one by one. This made no difference to the issue.

I have no idea what all this means but I have been able to consistently reproduce this in several experiments this morning. In other words, the Notes icon is only visible (a) in Netlify and (b) when launching from the dock using a Safari link.
Meanwhile, the category and month note icons remain visible in all these experiments!

**Comment by Kidglove57** [comment]
IMPORTANT UPDATE TO PREVIOUS POST:
I think I have a breakthrough. As background to this, I always the URL that I have saved always launches Actual to open at my Budget screen. When I do this the Accounts Note icon cannot be seen.

Quite by accident I just realised that if the URL that I use is specifically set to (say) "www.xxx.net/accounts" then I can see the Notes icon after launch (this was why the URL that I had dragged to the dock worked!)

**Comment by MikeBishop** [comment]
I'm seeing something similar with Brave/Windows.  If I click my usual shortcut (to `/budget`) and then click to the account with the note, the icon isn't there.  If I manually type the `/accounts` URL and then click to the same account, the note does display.

**Comment by j-f1** [comment]
This is a CSS issue where the `opacity: 1` is being overridden by `opacity: 0` on the parent.

**Comment by Kidglove57** [comment]
Thank you @j-f1. Much appreciated and good to know that those of us experiencing are not simply outliers.

**Comment by j-f1** [comment]
Can you try with https://deploy-preview-700--actualbudget.netlify.app/ and let me know if things are working for you?

**Comment by Kidglove57** [comment]
> Can you try with https://deploy-preview-700--actualbudget.netlify.app/ and let me know if things are working for you?

Thank you @j-f1. Yes, they are. Although in fairness they were working in Netlify previously. Just not in "real life".
